### PR TITLE
SL x snowflake guide

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr enhances the existing SL guide by adding more detailed steps for users. add'l steps mirror existing quickstars and include:
- set up new warehouse
- load data to warehouse
- create cloud acct
- build project
- create semantic models
-define metrics.
...etc.

this is in collaboration with the training team. [notion draft here](https://www.notion.so/dbtlabs/First-draft-5f46cc9d2ba14e17bec87fb4d7416112) and [notion project](https://www.notion.so/dbtlabs/dbt-SL-quickstart-refresh-zero-to-hero-workshop-583f3c8f8e7a4238802e5ac4cbc4712d?pvs=4)

### CODE REVIEW

File: website/docs/guides/quickstart-sl.md [hunk 1]
```diff
@@ -0,0 +1,862 @@
...
+  - Choose a cloud provider and region: Before proceeding, you will need to select a cloud provider. For this setup, all cloud providers and regions will work so choose whichever you’d like.
+- Basic understanding of SQL and dbt (meaning you either used dbt before or completed the [dbt Fundamentals](https://courses.getdbt.com/collections) course).
+
+## Create new Snowflake worksheet and set up environment
+
+1. Log in to your trial Snowflake account.
+2. In the Snowflake user interface (UI), click **+ Worksheet** in the upper right corner.
+3. Select **SQL Worksheet** to create a new worksheet.
+
+ADD IMAGE HERE
```
Comment (mirnawong1) [inline website/docs/guides/quickstart-sl.md]: adding image

File: website/docs/guides/quickstart-sl.md [hunk 2]
```diff
@@ -0,0 +1,862 @@
...
+    description: "percent of orders that are large"
+    type: derived
+    type_params:
+      expr: large_orders/order_count
+      metrics:
+        - name: large_orders
+        - name: order_count
+```
+
+## Add second semantic model to your project
```
Comment (mirnawong1) [inline website/docs/guides/quickstart-sl.md]: should this be a challenge? the guide has 16 steps and wondering if we can turn this into a challenge  via a toggle to reduce the number of steps?

File: website/snippets/_sl-connect-and-query-api.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
-    * Refer to the [SQL query syntax](/docs/dbt-cloud-apis/sl-jdbc#querying-the-api-for-metric-metadata) to query metrics using the API.  
-- To learn more about the sophisticated integrations that connect to the dbt Semantic Layer, refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more info. You can also use [Exports](/docs/use-dbt-semantic-layer/exports) to expose tables of metrics and dimensions in your data platform and create a custom integration with tools such as PowerBI, and more.
+To query your metrics using Google Sheets:
+1. Make sure you have a Gmail account. If you don’t already have one, make one at [gmail.com](http://gmail.com/).
+2. To set up Google Sheets and query your metrics, follow the detailed instructions on the [Google Sheets integration](/docs/use-dbt-semantic-layer/gsheets) page.
+3. Start playing around with it! Query a metric, like `order_total`. Filter it with a dimension, like `order_date`. You can also use the `group_by` parameter to group your metrics by a specific dimension.
```
Comment (mirnawong1) [inline website/snippets/_sl-connect-and-query-api.md]: can we have a screenshot or gif here?

File: website/snippets/_sl-test-and-query-metrics.md [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
 
-You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. Alternatively, you can run commands with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or with SQL client tools like DataGrip, DBeaver, or RazorSQL.
+You can use the **Preview** or **Compile** buttons in the IDE, or use the DAG for semantic validations and make sure your metrics are defined. Alternatively, you can run commands with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or with SQL client tools like DataGrip, DBeaver, or RazorSQL.
 
 :::
 
 <Tabs>
 
-<TabItem value="cloud" label="dbt Cloud">
+<TabItem value="cloudide" label="dbt Cloud IDE">
```
Comment (mirnawong1) [inline website/snippets/_sl-test-and-query-metrics.md]: added ide tab

File: website/docs/guides/sl-snowflake-qs.md [hunk 5]
```diff
@@ -0,0 +1,877 @@
...
+The dbt Semantic Layer, powered by [MetricFlow](/docs/build/about-metricflow), simplifies setting up key business metrics. It centralizes definitions, avoids duplicate code, and ensures easy access to metrics in downstream tools. MetricFlow helps manage company metrics with ease, allowing you to define metrics in your dbt project and query them in dbt Cloud with [MetricFlow commands](/docs/build/metricflow-commands).
+
+
+import SLCourses from '/snippets/_sl-course.md';
+
+<SLCourses/>
+
+This quickstart guide is designed for dbt Cloud users using Snowflake as their data platform. It focuses on building and defining metrics, setting up the dbt Semantic Layer in a dbt Cloud project, and querying metrics in Google Sheets.
+
+If you're using a data platform other than Snowflake, this guide is also applicable to you. To adapt the setup for your specific platform, follow the account setup and data loading instructions detailed on the following pages:
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
If you're using a data platform other than Snowflake, this guide is also applicable to you. You can adapt the setup for your specific platform by following the account setup and data loading instructions detailed on the following pages:
```

File: website/docs/guides/sl-snowflake-qs.md [hunk 6]
```diff
@@ -0,0 +1,877 @@
...
+- Set up dbt Semantic Layer in dbt Cloud
+- Connect and query metrics with Google Sheets
+
+## Create new Snowflake worksheet and set up environment
+
+1. Log in to your trial Snowflake account.
+2. In the Snowflake user interface (UI), click **+ Worksheet** in the upper right corner.
+3. Select **SQL Worksheet** to create a new worksheet.
+
+ADD IMAGE HERE
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
```

File: website/docs/guides/sl-snowflake-qs.md [hunk 7]
```diff
@@ -0,0 +1,877 @@
...
+
+### Set up Snowflake environment
+
+The data used here is stored as CSV files in a public S3 bucket and the following steps will guide you through how to prepare your Snowflake account for that data and upload it.
+
+1. Create a new virtual warehouse, two new databases (one for raw data, the other for future dbt development), and two new schemas (one for `jaffle_shop` data, the other for stripe data). To do this, run the following SQL commands one by one. Type them into the Editor of your new Snowflake SQL worksheet to set up your environment.
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
Create a new virtual warehouse, two new databases (one for raw data, the other for future dbt development), and two new schemas (one for `jaffle_shop` data, the other for stripe data).

1. Run the following SQL commands one by one by typing them into the Editor of your new Snowflake SQL worksheet to set up your environment.
```

File: website/snippets/_sl-test-and-query-metrics.md [hunk 8]
```diff
@@ -1,21 +1,20 @@
-This section explains how you can test and run MetricFlow commands with dbt Cloud or dbt Core (dbt Cloud IDE support coming soon). dbt Cloud IDE users can skip to [Run a production job](#run-a-production-job) to run a model.
+To work with metrics in dbt, you have several tools to validate or run commands. Here's how you can test and query metrics depending on your setup:
 
-:::important Testing and querying metrics in the dbt Cloud IDE is currently not supported
+- [**dbt Cloud IDE users**](#dbt-cloud-ide-users) &mdash; Currently, running MetricFlow commands directly in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) isn't supported, but it's on the way! You can still validate metrics using the **Preview** or **Compile** options, or visually through the DAG for semantic checks. This ensures your metrics are correctly defined without directly running commands.
+- [**dbt Cloud CLI users**](#dbt-cloud-cli-users) &mdash; The [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) enables you to run MetricFlow commands for direct interaction with metrics.
```
Comment (mirnawong1) [inline website/snippets/_sl-test-and-query-metrics.md]:
```suggestion
- [**dbt Cloud CLI users**](#dbt-cloud-cli-users) &mdash; The [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) enables you to run [MetricFlow commands](/docs/build/metricflow-commands#metricflow-commands) for direct interaction with metrics.
```

File: website/snippets/_sl-test-and-query-metrics.md [hunk 9]
```diff
@@ -1,21 +1,20 @@
...
+- **dbt Core users** &mdash; Use the MetricFlow CLI for command execution. Whicle this guide focuses on dbt Cloud users, dbt Core users can find detailed MetricFlow CLI instructions in the [MetricFlow commands](/docs/build/metricflow-commands#metricflow-commands) page.
```
Comment (mirnawong1) [inline website/snippets/_sl-test-and-query-metrics.md]:
```suggestion
- **dbt Core users** &mdash; Use the MetricFlow CLI for command execution. Whicle the this guide focuses on dbt Cloud users, dbt Core users can find detailed MetricFlow CLI setup instructions in the [MetricFlow commands](/docs/build/metricflow-commands#metricflow-commands) page. Note that to use the dbt Semantic Layer, you need to have a Team or Enterprise account.
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 10]
```diff
@@ -36,9 +36,9 @@ plan="dbt Cloud Team or Enterprise"
 <div className="grid--3-col">
 
 <Card
-    title="Get started with the dbt Semantic Layer and MetricFlow"
-    body="Build and define metrics with MetricFlow, set up the dbt Semantic Layer, and query them using the Semantic Layer API."
-    link="/docs/use-dbt-semantic-layer/quickstart-sl"
+    title="Quickstart with the dbt Cloud Semantic Layer"
+    body="Build and define metrics with MetricFlow, set up the dbt Semantic Layer, and query them using our first-class integrations."
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md]:
```suggestion
    body="Build and define metrics, set up the dbt Semantic Layer, and query them using our first-class integrations."
```

File: website/docs/docs/use-dbt-semantic-layer/sl-faqs.md [hunk 11]
```diff
@@ -136,7 +136,7 @@ MetricFlow is hosted in dbt Cloud. Requests from the [Semantic Layer APIs](/docs
 2. Then you build your metrics on top of these semantic models. This is all done in `.yml` configurations alongside your dbt models in your projects. 
 3. Once you've defined your metrics and semantic models, you can [configure the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) in dbt Cloud.
 
-Read our [Quickstart](/docs/use-dbt-semantic-layer/quickstart-sl) for more information.
+Read our [Quickstart](/guides/sl-snowflake-qs) for more information.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/sl-faqs.md]:
```suggestion
Read our [dbt Semantic Layer quickstart](/guides/sl-snowflake-qs) guide for more information.
```

File: website/docs/guides/sl-snowflake-qs.md [hunk 12]
```diff
@@ -0,0 +1,878 @@
...
+
+import DeprecationNotice from '/snippets/_sl-deprecation-notice.md';
+
+<DeprecationNotice />
+ 
+ </VersionBlock>
+
+## Introduction
+
+The dbt Semantic Layer, powered by [MetricFlow](/docs/build/about-metricflow), simplifies setting up key business metrics. It centralizes definitions, avoids duplicate code, and ensures easy access to metrics in downstream tools. MetricFlow helps manage company metrics with ease, allowing you to define metrics in your dbt project and query them in dbt Cloud with [MetricFlow commands](/docs/build/metricflow-commands).
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), powered by [MetricFlow](/docs/build/about-metricflow), simplifies setting up key business metrics. It centralizes definitions, avoids duplicate code, and ensures easy access to metrics in downstream tools. MetricFlow helps manage company metrics with ease, allowing you to define metrics in your dbt project and query them in dbt Cloud with [MetricFlow commands](/docs/build/metricflow-commands).
```

File: website/docs/guides/sl-snowflake-qs.md [hunk 13]
```diff
@@ -0,0 +1,878 @@
...
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
+
+https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_new-sl-setup.md
+-->
+
+<SlSetUp/>
+
+## Connect and query with Google Sheets
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository)
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md:905]: this isn't something we do but i've added a comment on the snippet location to help future contributions find the snippet when trying to propose changes.

File: website/docs/guides/sl-snowflake-qs.md [hunk 14]
```diff
@@ -0,0 +1,875 @@
...
+    label: "pct_of_orders_that_are_large"
+    description: "percent of orders that are large"
+    type: derived
+    type_params:
+      expr: large_orders/order_count
+      metrics:
+        - name: large_orders
+        - name: order_count
+```
+
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md:807]: notes for me: might be good to add a screenshot to show what success looks like

Screenshot 2024-04-12 at 17 15 52

File: website/docs/guides/sl-snowflake-qs.md [hunk 15]
```diff
@@ -0,0 +1,875 @@
...
+Let’s expand your project's analytical capabilities by adding another semantic model in your other marts model: `dim_customers.yml`.
+
+After setting up your orders model:
+
+1. Create the file `models/metrics/dim_customers.yml`.
+2. Copy the following query into the file and click **Save**.
+
+```yaml
+semantic_models:
+  - name: dim_customers
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]: i thnk this should be 'customers' not dim right @MichelleBaird-DBT ?

File: website/docs/guides/sl-snowflake-qs.md [hunk 16]
```diff
@@ -0,0 +1,875 @@
...
+    defaults:
+      agg_time_dimension: most_recent_order_date
+    description: |
+      semantic model for dim_customers
+    model: ref('dim_customers')
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]: same here (customers not dim)

File: website/docs/guides/sl-snowflake-qs.md [hunk 17]
```diff
@@ -0,0 +1,875 @@
...
+
+## Test and query metrics
+
+<!-- The below snippets (or reusables) can be found in the following file locations in the docs code repository) 
+
+https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/snippets/_sl-test-and-query-metrics.md
+-->
+
+<TestQuery />
+
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md:874]: i think before this, we should make it clear the user has to commit changes and merge the pr.

File: website/docs/guides/sl-snowflake-qs.md (hunk 15 above)
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
  - name: customers
```

File: website/docs/guides/sl-snowflake-qs.md [hunk 18]
```diff
@@ -0,0 +1,875 @@
...
+2. Copy the following query into the file and click **Save**.
+
+```yaml
+semantic_models:
+  - name: customers
+    defaults:
+      agg_time_dimension: most_recent_order_date
+    description: |
+      semantic model for dim_customers
+    model: ref('dim_customers')
```
Comment (mirnawong1) [inline website/docs/guides/sl-snowflake-qs.md]:
```suggestion
    model: ref('customers')
```

File: website/snippets/_sl-test-and-query-metrics.md [hunk 19]
```diff
@@ -1,21 +1,20 @@
...
-Support for running [MetricFlow commands](/docs/build/metricflow-commands) in the dbt Cloud IDE is not available but is coming soon. 
+Alternatively, you can run commands with SQL client tools like DataGrip, DBeaver, or RazorSQL.
 
-You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. Alternatively, you can run commands with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or with SQL client tools like DataGrip, DBeaver, or RazorSQL.
+### dbt Cloud IDE users
 
-:::
-
-<Tabs>
+You can validate your metrics in the dbt Cloud IDE by selecting the metric you want to validate and viewing it in the **Lineage** tab. (Support for running commands in the dbt Cloud IDE support coming soon).
```
Comment (mirnawong1) [inline website/snippets/_sl-test-and-query-metrics.md]:
```suggestion
You can validate your metrics in the dbt Cloud IDE by selecting the metric you want to validate and viewing it in the **Lineage** tab. (Support for running commands in the dbt Cloud IDE support coming soon).

Once validated, make sure you commit and merge your changes in your project. 
```

File: website/snippets/_sl-run-prod-job.md [hunk 20]
```diff
@@ -1,7 +1,9 @@
 Once you’ve defined metrics in your dbt project, you can perform a job run in your deployment environment in dbt Cloud to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer currently.
```
Comment (mirnawong1) [inline website/snippets/_sl-run-prod-job.md]:
```suggestion
Once you’ve committed and merged your metric changes in your dbt project, you can perform a job run in your deployment environment in dbt Cloud to materialize your metrics. The deployment environment is only supported for the dbt Semantic Layer currently. 
```

File: website/docs/docs/build/sl-getting-started.md [hunk 21]
```diff
@@ -1,94 +0,0 @@
...
-
-However, to experience the power of the universal [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and query those metrics in downstream tools, you'll need a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. 
-
-import SLCourses from '/snippets/_sl-course.md';
-
-<SLCourses/>
-
-## Prerequisites
-
-import SetUp from '/snippets/_v2-sl-prerequisites.md';
```
Comment (nghi-ly) [inline website/docs/docs/build/sl-getting-started.md:39]: is this snippet used elsewhere? if not, maybe delete it for cleanup?

File: website/docs/guides/sl-snowflake-qs.md [hunk 22]
```diff
@@ -0,0 +1,870 @@
...
+This quickstart guide is designed for dbt Cloud users using Snowflake as their data platform. It focuses on building and defining metrics, setting up the dbt Semantic Layer in a dbt Cloud project, and querying metrics in Google Sheets.
+
+If you're using a data platform other than Snowflake, this guide can also be applicable to you. You can adapt the setup for your specific platform by following the account setup and data loading instructions detailed on the following pages:
+
+- [BigQuery](https://docs.getdbt.com/guides/bigquery?step=2) (Steps 2 - 5)
+- [Databricks](https://docs.getdbt.com/guides/databricks?step=2) (Steps 2 - 4)
+- [Microsoft Fabric](https://docs.getdbt.com/guides/microsoft-fabric?step=2) (Steps 2 - 3)
+- [Redshift](https://docs.getdbt.com/guides/redshift?step=2) (Steps 2 - 4)
+- [Starburst Galaxy](https://docs.getdbt.com/guides/starburst-galaxy?step=2) (Steps 2 - 5)
+
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md:47]: consider changing how this ^ list is constructed. maybe linkout to the steps/sections?

- the "(Steps x - y)" strings become a maintenance burden
- the reader has to flip back and forth between tabs/windows

**Review by nghi-ly — COMMENTED** [review]
@mirnawong1 : done reviewing the first 3 step pages

File: website/docs/docs/build/sl-getting-started.md (hunk 21 above)
Comment (mirnawong1) [reply website/docs/docs/build/sl-getting-started.md:39]: yes i believe it is in the 'setup sl' ones : https://docs.getdbt.com/docs/use-dbt-semantic-layer/setup-sl

File: website/docs/guides/sl-snowflake-qs.md [hunk 23]
```diff
@@ -0,0 +1,870 @@
...
+  ```sql
+  create table raw.jaffle_shop.customers
+  ( id integer,
+    first_name varchar,
+    last_name varchar
+  );
+  ```
+
+  You should see a ‘Table `CUSTOMERS` successfully created.’ message.
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md:184]: consider removing this line. seems low value and superfluous (for conciseness and faster scanning)

File: website/docs/guides/sl-snowflake-qs.md [hunk 24]
```diff
@@ -0,0 +1,870 @@
...
+  copy into raw.jaffle_shop.customers (id, first_name, last_name)
+  from 's3://dbt-tutorial-public/jaffle_shop_customers.csv'
+  file_format = (
+      type = 'CSV'
+      field_delimiter = ','
+      skip_header = 1
+      );
+  ```
+
+  You should see a confirmation message after running the command.
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md:198]: same as prev comment. 

consider removing lines like this. seems low value and superfluous (for conciseness and faster scanning)

File: website/docs/guides/sl-snowflake-qs.md [hunk 25]
```diff
@@ -0,0 +1,870 @@
...
+-- Create two databases: one for raw data and another for analytics
+create database raw;
+create database analytics;
+
+-- Within the 'raw' database, create two schemas: 'jaffle_shop' and 'stripe'
+create schema raw.jaffle_shop;
+create schema raw.stripe;
+```
+
+## Load data into Snowflake
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md:171]: this whole step page is very similar to the Snowflake QS. 

ideas:
- use a snippet/partial (single source) to relieve maintenance burden
- add as a prereq that "Snowflake QS guide must be completed".

**Review by nghi-ly — COMMENTED** [review]
@mirnawong1 : done reviewing the step pages 4-7

File: website/docs/guides/sl-snowflake-qs.md [hunk 26]
```diff
@@ -0,0 +1,870 @@
...
+    description: |
+      Order fact table. This table’s grain is one row per order.
+    model: ref('fct_orders')
+
+```
+
+The following section will explain [dimensions](/docs/build/dimensions), [entities](/docs/build/entities), and [measures](/docs/build/measures) in more detail, showing how they each play a role in semantic models.
+
+- [Entities](#entities) act as unique identifiers (like ID columns) that link data together from different tables.
+- [Dimensions](#dimensions) allow us to categorize and filter data, making it easier to organize.
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md]: unclear who is `us` in this bullet

File: website/docs/guides/sl-snowflake-qs.md [hunk 27]
```diff
@@ -0,0 +1,870 @@
...
+- [Measures](#measures) help us calculate data, providing valuable insights through aggregation.
```
Comment (nghi-ly) [inline website/docs/guides/sl-snowflake-qs.md]: same thing here

**Review by nghi-ly — COMMENTED** [review]
@mirnawong1 : done reviewing step pages 8 - 10

**Review by nghi-ly — COMMENTED** [review]
@mirnawong1 : done reviewing step pages 11 - 13

File: website/snippets/_new-sl-setup.md [hunk 28]
```diff
@@ -3,17 +3,13 @@ You can set up the dbt Semantic Layer in dbt Cloud at the environment and projec
 - You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
     * Enterprise plan &mdash; Developer license with Account Admin permissions. Or Owner with a Developer license, assigned Project Creator, Database Admin, or Admin permissions.
     * Team plan &mdash; Owner with a Developer license.
+    * If you are using a free trial dbt Cloud account, you are on a trial of the Team plan as an Owner, so you're good to go.
 - You must have a successful run in your new environment.
```
Comment (nghi-ly) [inline website/snippets/_new-sl-setup.md:7]: this reads like Prereqs. consider moving this info to step page 2. that way all the requirements are grouped together

**Review by nghi-ly — COMMENTED** [review]
@mirnawong1 : done with the last 3 step pages (14 - 16)

**Review by nghi-ly — APPROVED** [review]
fantastic work! thank you!

File: website/docs/guides/sl-snowflake-qs.md (hunk 22 above)
Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:47]: good point!

Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:47]: it's prob likely that some tab back and forth might happen (maybe fetchin the (step 2-5) content so the user never leaves the page is a longer term experiment). but I've used tabs for this

Screenshot 2024-04-19 at 12 36 50

File: website/docs/guides/sl-snowflake-qs.md (hunk 23 above)
Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:184]: totally understand your point, however when i went through the training, i feel in limbo not knowing what success looked like and this helps the user validate their steps, which training agreed with too.

Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:184]: going to resolve the other suggestions as i have the same response to them 🙏

File: website/docs/guides/sl-snowflake-qs.md (hunk 25 above)
Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:171]: thanks Ly! this is teh same as the guide - will open an issue to create a partial!

Comment (mirnawong1) [reply website/docs/guides/sl-snowflake-qs.md:171]: https://github.com/dbt-labs/docs.getdbt.com/issues/5310

**Pull request merged**